healthcheck.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Run a curl against the Jenkins instance installed in a Dockerfile
+# to do a basic health check
+
+set -e
+set -x
+
+echo "Wait 30 seconds for Jenkins to boot"
+sleep 30
+
+echo "Curling against the Jenkins server"
+STATUS_CODE=$(curl -sL -w "%{http_code}" localhost:8080 -o /dev/null)
+
+if [[ "$STATUS_CODE" == "200" ]]; then
+    echo "Jenkins has come up and ready to use"
```

Comment:
Super nit: This should probably say "Jenkins has come up and **is** ready to use"